README.md

```diff
@@ -9,41 +9,61 @@ So to launch the server, it will only start the server.
 Mods and bots are managed at startup by checking your `config.yml`.
 `npm` is only invoked if changes are made to your `config.yml`.
 
-## Usage
+## Installation (Recommended)
+**Step 1: Clone/Download this repository.**
 
-The recommended way to use this image is through docker compose.
-An example setup with [screepsmod-mongo] can be seen in the compose file [docker-compose.yml](docker-compose.yml) in this repo.
-This spins up a server with a mongo and redis service beside it.
-Remember that `screepsmod-mongo` must be in the list of mods in your `config.yml`
+**Step 2:  Download Docker Desktop** 
 
-Copy the compose file and run `docker compose up`. You should see the services starting.
-You can now access your private screeps server on `http://localhost:21025`
+- [Download For Mac (Intel)](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=module) 
+- [Download For Mac (Apple Silicon)](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=module)
+- [Download For Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=module)
+- [Download For Linux](https://docs.docker.com/desktop/linux/install/)
```

Comment:
I would prefer just linking to Docker Desktop, and let their site handle redirecting the user to the correct thing.

```suggestion
- [Download](https://www.docker.com/products/docker-desktop/)
```